tests/Propel/Tests/Generator/Command/InitCommandTest.php

```diff
@@ -48,6 +48,10 @@ public function setUp()
 
     public function testExecute()
     {
+        if (!method_exists(CommandTester::class, 'setInputs')) {
+            $this->markTestSkipped('Interactive console input was not present in some earlier versions of symphony/console');
+        }
```

Comment:
This test is testing an interactive console prompt that can only be tested by passing data to the setInputs method, because setInputs is part of the CommandTester class that will create an actual `stream` for the data. You can't pass it as array params.

I attempted to write a non-interactive (eg: just arguments) version in c07a2d33 but this was silly because it means I'd still have to skip the next test below (`testExecuteAborted`) because there's no way to test if the command will get aborted.

I suggest a dedicated feature gets introduced to pass commandline args instead of interactive.